ui/packages/catalog-ui-search/src/main/webapp/react-component/dropdown/dropdown.js

```diff
@@ -231,4 +259,13 @@ class Dropdown extends React.Component {
   }
 }
 
+Dropdown.propTypes = {
```

Comment:
Do Poller and DropdownArea need PropTypes?

Reply:
They are internal implementation details so I don't think it would add any value.